src/ConnectedMode/Migration/MigrationStrings.resx

```diff
@@ -136,7 +136,8 @@
     <value>[Migration] Deleting the legacy .sonarlint...</value>
   </data>
   <data name="ErrorDuringMigation_NormalLog" xml:space="preserve">
-    <value>[Migration] Error during migration: {0}</value>
+    <value>[Migration] Error during migration: {0}
+  Run migration again with verbose logging enabled for more information.</value>
```

Comment:
I added the second line so users know to get more info - it could simplify responding to any forum questions.